lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
```

Comment:
The idea is if there are multi kegs installed, we need to choose one and use the build options of that one as the args. 

To choose, we first use the linked keg. But if none of them is linked, I would select the latest version. Hence, we require `version` to do the comparison.